youtrack_mcp/api/issues.py

```
@@ -217,4 +217,180 @@ def add_comment(self, issue_id: str, text: str) -> Dict[str, Any]:
+        try:
+            # Get all issue link types with their names and directions
+            fields = "id,name,directed,sourceToTarget,targetToSource,aggregation,readOnly"
+            params = {"fields": fields}
+            link_types = self.client.get("issueLinkTypes", params=params)
+            logger.debug(f"Retrieved {len(link_types) if isinstance(link_types, list) else 0} link types")
+            return link_types
+        except Exception as e:
+            logger.error(f"Failed to get available link types: {str(e)}")
+            raise
+    
+    def update_issue(self, issue_id: str, **fields) -> Dict[str, Any]:
```

Ruff (0.11.9)

Redefinition of unused `update_issue` from line 146

(F811)